core/src/test/scala/github/ParseGithubOpenApiTest.scala

```diff
@@ -0,0 +1,16 @@
+package net.zhenglai
```

Comment:
Please use https://github.com/github/rest-api-description/tree/main/descriptions/api.github.com swagger file, this is for the official github api, and ghes is for github enterprise server I guess, not in our current scope.

Or please correct me if I am wrong